source/developer/mobile-developer-setup.rst

```diff
@@ -212,99 +212,653 @@ In order to develop and build the Mattermost mobile apps you'll need to get a co
+   select the **login** keychain and **My Certificates** from the side menu.
+ .. image:: ../../source/images/mobile/ios_keychain_select.png
+
+ - Find the certificate you imported and then right click to **export** it as a **.p12** file
+ .. image:: ../../source/images/mobile/ios_keychain_export.png
+
+ - Enter a name for the filename and click **Save**
+ .. image:: ../../source/images/mobile/ios_keychain_export_save.png
+
+ - Leave the **password** blank and then click **OK**
+ .. image:: ../../source/images/mobile/ios_keychain_export_password.png
+
+ - Convert the downloaded certificate to **.pem**
+ .. code-block:: bash
+
+   $ openssl x509 -in aps.cer -inform DER -out aps_production.pem
+
+ - Extract the private key from the certificate you exported
+ .. code-block:: bash
+
+   $ openssl pkcs12 -in Certificates.p12 -out aps_production_priv.pem -nodes -clcerts -passin pass:
+
+ - Verify the certificate works with apple
+ .. code-block:: bash
+
+   $ openssl s_client -connect gateway.push.apple.com:2195 -cert aps_production.pem -key aps_production_priv.pem
+
+ - Copy the private key file ``aps_production_priv.pem`` into your ``mattermost-push-proxy/config`` directory
+
+ - Open the **mattermost-push-proxy.json** file under the ``mattermost-push-proxy/config`` directory and add the path to the private key file
+   as the value for **"ApplePushCertPrivate"** and the value for **"ApplePushTopic"** with your *Bundle Identifier*
+ .. image:: ../../source/images/mobile/proxy-config.png
+
+- Finally, restart your Mattermost Push Proxy server, and your app should start receiving push notifications.
+
+Overriding assets & White labeling
+-----------------------------------
+
+Sometimes you end up wanting to white label the Mattermost app with your organization assets to have a sense of ownership. With this in mind
+we've made the mobile apps easy to white label and to override the assets used, the downside is that you have to `Build your own app from source`_.
+
+Ok so let's understand how this work!
+
+If you take a look at the `Project Directory Structure`_ you'll see that there is an assets folder, in this folder you have the base directory which is the assets that we provide,
+they include localization files, images and a release folder that contains the icons of the app to be used when building into release mode for deployment (this last set of assets is optional).
+
+If you want to replace the assets used in the project with your own assets all you need to do is create a sub-directory called ``override`` under the ``assets`` folder, and using the same
+sub-directory structure as the ``base`` directory as well as the same file names, you can have your own assets take be used instead.
+
+Localization strings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To replace some or all of the strings in the app in any supported lenguage, copy over the files for the languages that you want to override under ``assets/base/i18n`` to ``assets/override/i18n``
```

Comment:
They only need to include strings that they want to change in the overridden version. If they don't provide a string, it always uses the base one. I'll update this section and remove the warning

Reply:
Awesome thanks HH